panther_manager/include/panther_manager/plugins/action/call_set_bool_service_node.hpp

```diff
@@ -1,26 +1,31 @@
 #ifndef PANTHER_MANAGER_CALL_SET_BOOL_SERVICE_NODE_HPP_
 #define PANTHER_MANAGER_CALL_SET_BOOL_SERVICE_NODE_HPP_
 
+#include <string>
+
+#include <behaviortree_cpp/basic_types.h>
+#include <behaviortree_cpp/tree_node.h>
+
```

Comment:
```suggestion

#include <ros/console.h>

```

Reply:
I believe you meant to add this in `call_set_bool_service_node.cpp` file

Reply:
Yes. I have no idea why it ended up here